native/packages/react-native-bpk-component-card/stories.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react-native';
+import { StyleSheet, View, Platform } from 'react-native';
+import BpkCard from './index';
+import BpkText from '../react-native-bpk-component-text';
+
+
+const tokens = Platform.select({
+  ios: () => require('bpk-tokens/tokens/ios/base.react.native.common.js'), // eslint-disable-line global-require
+  android: () => require('bpk-tokens/tokens/android/base.react.native.common.js'), // eslint-disable-line global-require
+})();
+
+const {
+  spacingSm,
+  colorGray50,
+  colorRed500,
+  colorGreen500,
+  colorYellow500,
+  colorBlue700,
+  colorGray500,
+} = tokens;
+
+const styles = StyleSheet.create({
+  centered: {
+    flex: 1,
+    flexDirection: 'column',
+    justifyContent: 'center',
+    padding: spacingSm,
```

Comment:
Can we make sure that the padding around stories is consistent across components (especially for screenshots) - see https://github.com/Skyscanner/backpack/blob/master/native/packages/react-native-bpk-component-text/stories.js#L12

Reply:
sure, will update the story and the screenshots as well